counters/counter.go

```diff
@@ -0,0 +1,39 @@
+package counters
+
+import (
+	"fmt"
+	"sort"
+
+	m "github.com/whatsapp/models"
+)
+
+type Counter interface {
```

Comment:
if you are not using this interface, so you don't need it (YAGNI)... remember that in Go the client of the api is the one interested in the definition of the interface, not the provider of the api